code/src/components/Styles/thoughtform.css

```diff
@@ -0,0 +1,47 @@
+/* Style for the form */
+.thoughts-form {
+    margin-top: 20px;
+    padding: 20px;
+    text-align: left;
+    background: #eeeeee;
+    border: 1px solid;
+    box-shadow: 5px 10px;
+    width: 100%;
+}
+
+.thoughts-form__input {
+    width: 100%;
+    padding: 5px 0px 80px 5px;
+
```

Comment:
u could remove the line of whitespace and when there is a 0 value you can omit the px :)